src/ansi-c/ansi_c_convert_type.cpp

```diff
@@ -13,6 +13,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <util/c_types.h>
 #include <util/config.h>
+#include <util/expr_util.h>
```

Comment:
Is that include still needed?

Reply:
Yep, we use `has_subexpr` in this module now.